.github/workflows/sim.yaml

```diff
@@ -0,0 +1,91 @@
+name: sim
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
```

Comment:
I doubt running these tests every PR. Think of `zetaclient` PRs - most of the time they don't touch `zetacore` part. Probably we should do the following:
- for every PR if `x/...` files were touched
- if a PR has `SIM_TESTS` label

Otherwise the CI will become even slower